src/operations/triggers.ts

```diff
@@ -44,8 +44,8 @@ export function createTrigger(mOptions: MigrationOptions) {
       operation,
       deferrable,
       deferred,
-      functionArgs = []
-    }: TriggerOptions = triggerOptions;
+      functionParams = []
```

Comment:
Making it consistent with the type and with usage on other places. It is a breaking change. Needs to be mentioned in release notes!